pkg/chartutil/requirements.go

```diff
@@ -216,6 +218,45 @@ func ProcessRequirementsTags(reqs *Requirements, cvals Values) {
 
 }
 
+func copyChartAsAlias(charts []*chart.Chart, dependentChart, aliasChart string) *chart.Chart {
+	var chartFound *chart.Chart
+	for _, existingChart := range charts {
+		if existingChart == nil {
+			continue
+		}
+		if existingChart.Metadata == nil {
+			continue
+		}
+		if existingChart.Metadata.Name != dependentChart {
+			continue
+		}
+
+		chartFound = new(chart.Chart)
+		chartFound.Metadata = &chart.Metadata{
```

Comment:
Because everything with the chart is pointers, you could easily shallow copy the `existingChart` and then shallow copy `existingChart.Metadata` so as to not have to manually copy every field